sssom/context.py

```diff
@@ -17,7 +17,7 @@
 # EXTERNAL_CONTEXT_PATH = HERE / "sssom.external.context.jsonld"
 
 SSSOM_URI_PREFIX = "https://w3id.org/sssom/"
-SSSOM_BUILT_IN_PREFIXES = ["sssom", "owl", "rdf", "rdfs", "skos", "semapv"]
+SSSOM_BUILT_IN_PREFIXES = ("sssom", "owl", "rdf", "rdfs", "skos", "semapv")
```

Comment:
Apparently this was getting overwritten after being assigned to `prefixes` in `utils.py`. So converting it into a tuple so it is immutable.